tests/validation/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@ set(VALIDATION_CUDA_SOURCES
 
 if(NMC_VALIDATION_DATA_DIR)
     if ("${CMAKE_VERSION}" MATCHES "^3.[78].")
-        message(WARNING "CMake ${CMAKE_VERSION} has broken FindCUDA; omitting NMC_DATADIR define.")
+        # disable warning
+        #message(WARNING "CMake ${CMAKE_VERSION} has broken FindCUDA; omitting NMC_DATADIR define.")
     else()
```

Comment:
We could move to a purely environment-variable based solution, but this would complicate running the validation tests (I think we'd need a wrapper script that CMake fills out that sets up the environment or we require the user supply the path on the validation test executable command line if the data directory is not found in a 'usual' relative location.)